### PR TITLE
Adding gluonts package

### PR DESCRIPTION
**Pull request by giswqs**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Relevant issue:
- https://github.com/autogluon/autogluon/issues/612

@arturdaraujo

### CODE REVIEW

**Comment by giswqs** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for review. Thanks.

**Comment by arturdaraujo** [comment]
One step closer

File: recipes/gluonts/meta.yaml
Comment (dhirschfeld) [inline recipes/gluonts/meta.yaml:49]: The copies of these files can be removed here.

By referencing them under `license_file` conda will automatically include copies from the source at build time.

**Comment by giswqs** [comment]
@dhirschfeld The source on PyPI does not include the NOTICE and LICENSE files, so I think they need to be included here. Otherwise, the recipe fails because of missing files. 

https://pypi.org/project/gluonts/

image

**Comment by giswqs** [comment]
@conda-forge-admin, please ping team

**Comment by giswqs** [comment]
@ocefpaf It is ready to be merged.

**Comment by dhirschfeld** [comment]
> The source on PyPI does not include the NOTICE and LICENSE files, so I think they need to be included here.

If the sdist doesn't include those files then you do need to include copies here, however that should probably be fixed upstream (though it doesn't need to hold up merging this PR)

File: recipes/gluonts/meta.yaml
Comment (dhirschfeld) [inline recipes/gluonts/meta.yaml:24]:
```suggestion
  host:
    - python
    - pip
```
AFAICS these aren't required to build the package?

`click` seems that it might be a run-time dependency?

**Comment by giswqs** [comment]
@dhirschfeld Thanks for the suggestion. I have made changes as suggested. 

Also, I have opened an issue on gluonts repo to suggest adding the LICENSE and NOTICE files in the next release. 

https://github.com/awslabs/gluonts/issues/2564

**Comment by dhirschfeld** [comment]
> Also, I have opened an issue on gluonts repo to suggest adding the LICENSE and NOTICE files in the next release.

That's the exact right thing to do! :+1:

Having a copy of the license here is fine whilst it's not included in the upstream sources so that shouldn't hold up merging this PR. It'll just require someone from @conda-forge/staged-recipes to do the final review/merge...

**Review by dhirschfeld — APPROVED** [review]
LGTM :shipit:

File: recipes/gluonts/meta.yaml
Comment (ocefpaf) [inline recipes/gluonts/meta.yaml:13]: You don't need to skip py2k in conda-forge. You may consider trying to make this noarch though by dropping dataclasses and py36.

Comment (giswqs) [reply recipes/gluonts/meta.yaml]: @ocefpaf Thanks for the suggestion. I have dropped dataclasses for py36 and changed to noarch.

**Comment by giswqs** [comment]
@conda-forge-admin, please ping team

It seems the `gluonts` and `autocfg` recipes have been stuck in the `staged-recipes` repo for >10 hours after being merged. Feedstocks are not being created.  Is this normal?

**Comment by arturdaraujo** [comment]
@giswqs probably something weird with python 3.11 integration, their site shows some "bot errors". I've checked a couple of times today a package that should've been updated early this morning and still nothing... 

https://conda-forge.org/status/

**Comment by giswqs** [comment]
@arturdaraujo Thanks for looking into that. Normally it takes less than an hour to create the feedstock once a recipe is accepted and merged. It has taken more than 12 hours this time. Something is not right.

**Pull request merged**
